homeassistant/components/onkyo/media_player.py

```diff
@@ -473,10 +458,41 @@ async def async_select_output(self, output):
         """Set hdmi-out."""
         self._update_avr("hdmi-output-selector", output)
 
+    def _update_avr(self, propname, value):
```

Comment:
Any reason to move `_update_avr` and `_query_avr`. I think it just complicates the diff

Reply:
Hmm, not really a reason I guess, good point :-P